### PR TITLE
PR template edits

### PR DESCRIPTION
**Pull request by kimsauce**
## Purpose of this pull request

This pull request updates and streamlines PR template.

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions, .clabot
- [x] Update Content - Revisions, updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, dependencies, new packages for the site (Docusaurus, Gatsby, React, etc.)

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -1,20 +1,17 @@
...
-- [ ] Yes, I've run `yarn install`
-- [ ] No, does not apply to me
-
 ## Purpose of this pull request
 
-This pull request...
+This pull request... <!-- brief description here -->
 
-<!-- Enter the GitHub Issue number or the Jira project and number (e.g., SUMO-12345) -->
+<!-- Enter the GitHub Issue number or the Jira ticket number (e.g., SUMO-12345) -->
```
Comment (jpipkin1) [inline .github/PULL_REQUEST_TEMPLATE.md:5]: Could you move this comment line about ticket numbers down to the section about tickets?

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -1,20 +1,17 @@
...
+<!-- What types of changes does your code introduce? Select the checkbox after clicking "Create pull request" button. -->
 
-- [ ] Minor Changes - Typos, formatting, slight revisions, .clabot
+- [ ] Minor Changes - Typos, formatting, slight revisions
 - [ ] Update Content - Revisions, updating sections
 - [ ] New Content - New features, sections, pages, tutorials
-- [ ] Site and Tools - Updates, maintenance, dependencies, new packages for the site (Docusaurus, Gatsby, React, etc.)
+- [ ] Site and Tools - .clabot, version updates, maintenance, dependencies, new packages for the site (Docusaurus, Gatsby, React, etc.)
+
+## Asana ticket (if applicable)
```
Comment (jpipkin1) [inline .github/PULL_REQUEST_TEMPLATE.md:15]:
```suggestion
## Ticket (if applicable)
```

**Review by jpipkin1 — CHANGES_REQUESTED** [review]
@kimsauce, I have just a couple of comments.

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 3]
```diff
@@ -1,20 +1,17 @@
...
-- [ ] Minor Changes - Typos, formatting, slight revisions, .clabot
+- [ ] Minor Changes - Typos, formatting, slight revisions
 - [ ] Update Content - Revisions, updating sections
 - [ ] New Content - New features, sections, pages, tutorials
-- [ ] Site and Tools - Updates, maintenance, dependencies, new packages for the site (Docusaurus, Gatsby, React, etc.)
+- [ ] Site and Tools - .clabot, version updates, maintenance, dependencies, new packages for the site (Docusaurus, Gatsby, React, etc.)
+
+## Ticket (if applicable)
+
+<!-- populates in this space automatically when you add this PR link to Asana ticket -->
```
Comment (kimsauce) [inline .github/PULL_REQUEST_TEMPLATE.md:17]:
```suggestion
<!-- enter your Jira, Asana, or GitHub ticket number -->
```

**Comment by kimsauce** [comment]
> @kimsauce, I have just a couple of comments.

thank u!

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (jpipkin1) [inline .github/PULL_REQUEST_TEMPLATE.md:5]:
```suggestion
```

Comment (jpipkin1) [reply .github/PULL_REQUEST_TEMPLATE.md:5]: You can remove this line since you have the "Ticket (if applicable)" section below.

**Review by jpipkin1 — APPROVED** [review]
Approved, @kimsauce, contingent on your addressing one more comment.

**Pull request merged**